sandbox-templates/react-markdown-with-create-react-app/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "react-markdown-with-create-react-app",
+  "version": "1.0.0",
+  "private": true,
+  "main": "src/index.jsx",
+  "dependencies": {
+    "react": "17.0.2",
+    "react-dom": "17.0.2",
+    "react-markdown": "7.1.0",
+    "react-scripts": "4.0.3"
+  },
+  "devDependencies": {
+    "typescript": "3.8.3"
```

Comment:
4.4?

Reply:
Updated to 4.4, and updated example to use tsx files